src/test/scala/org/apache/spark/sql/test/SharedSparkSessionGDAL.scala

```diff
@@ -23,7 +23,10 @@ trait SharedSparkSessionGDAL extends SharedSparkSession {
         val session = new TestSparkSession(conf)
         session.sparkContext.setLogLevel("FATAL")
         Try {
-            MosaicGDAL.prepareEnvironment(session, "/tmp")
+            val tempPath = Files.createTempDirectory("mosaic-gdal")
```

Comment:
@mjohns-databricks do we need this line?
Can we remove the comment as well?

Reply:
Hey, good call out. I wanted to get with you on testing this. I was thinking use the directory for a fuse install and then assert the number of artifacts are correct. There would be a handful of these for all the main combos.